apps/dashr-multipage-report/app.R

```diff
@@ -0,0 +1,1916 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(stringi)
+library(generator)
+library(plotly)
+library(random)
+library(data.table)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-multipage-report")
+
+supplyDemand <- fread("/Data/supplyDemand.csv")
+actualSeasonal <- fread("/Data/actualSeasonal.csv")
+industrailProd <- fread("/Data/industrailProd.csv")
+globalMarket <- fread("/Data/globalMarket.csv")
+oecdCommersial <- fread("/Data/oecdCommersial.csv")
+wtiPrices <- fread("/Data/wtiPrices.csv")
+epxEquity <- fread("/Data/epxEquity.csv")
+chinaSpr <- fread("/Data/chinaSpr.csv")
+oecdIndustry <- fread("/Data/oecdIndustry.csv")
+wtiOilprices <- fread("/Data/wtiOilprices.csv")
+productionCost <- fread("/Data/productionCost.csv")
+production2015 <- fread("/Data/production2015.csv")
+energyShare <- fread("/Data/energyShare.csv")
+
+set.seed(7685)
+front_phone <- r_phone_numbers(5, use_hyphens = T)
+front_email <- r_email_addresses(5)
+
+
+LipP <- function(n, l){
+  # function generates and lorem ipsum script takes input number of paragraphs (n)
+  # and a logical input (l) TRUE of FALSE  for start point of lorem TRUE being from 
+  # the beginining 
+  stri_rand_lipsum(nparagraphs = n, start_lipsum = l)
+}
+
+generate_table <- function(df, nrows=10)
+  # function generates a dash table from a supplied data frame (df)
+  #  and number of rows (nrows) to display 
+{
+  n <- min(nrows, nrow(df))
+  rows <- lapply(seq(1, n), function(i) {
+    htmlTr(children = lapply(as.character(df[i,]), htmlTd))
+  })
+  
+  header <- htmlTr(children = lapply(names(df), htmlTh))
+  
+  htmlTable(
+    children = c(list(header), rows)
+  )
+  
+}
+
+color_1 <- "#003399"
+color_2 <- "#00ffff"
+color_3 <- "#002277"
+color_b <- "#F8F8FF"
+
+
+app <- Dash$new(external_stylesheets = list("https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css",
```

Comment:
I think we're supposed to have the stylesheets in the assets folder for the example apps